core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/pcf",
-  "version": "0.4.0-beta.1",
```

Comment:
@akshay-madhukar I recall Jackson's parent child enhancement is on main while he created a release branch for PCF pre parent child.  I think we should backport the dependency changes back to the release branch and do this release build from the release branch.  We can do a beta build from the parent child stuff on main at a later date.